articles/native-platforms/ios-objc/08-calling-APIs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Calling your API
+description: This tutorial will show you how to use Auth0 to authenticate with your own API server.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'auth0-samples/auth0-ios-objc-sample',
+  pkgBranch: 'master',
+  pkgPath: '08-Calling-APIs',
+  pkgFilePath: '08-Calling-APIs/Auth0Sample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+### In the Beginning
+
+#### i. Be familiar with Auth0
+
+This tutorial assumes you are already familiar with Auth0 and how to Sign up and Login using Lock or Auth0 Toolkit. **If you're not sure, check out [this tutorial](01-login.md) first.**
+
+### 1. Purpose of this tutorial
+
+This tutorial will enable you to delegate all your authentication needs on Auth0, leaving your API server to handle only your core business processes.
+To achieve this, we will send the Auth0 token to the server through the `Authentication` header of the request, and the server at it's time will have to check with Auth0 servers to corroborate that this token is valid. We will not go into details about this later part, since there are other very good tutorials that cover the server side of this process. 
```

Comment:
- to handle → handling

- we will send the Auth0 token → we will send an Auth0 token 
*(there is no "the" chosen one token, it's just any of them)*

- Syntax fix: To achieve this, we will send the Auth0 token to the server through the `Authentication` header of the request. The server, at that moment, will have to check with Auth0 servers to corroborate that this token is valid.

- Please, add a link to at least one of those tutorials (check in my Swift equivalent tutorial, there is one)